components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/engine/runtime/xpath/JaxpFunctionResolver.java

```diff
@@ -194,18 +245,24 @@ public Object evaluate(List args) throws XPathFunctionException {
     /**
      * Returns the potential owners of the task. It MUST evaluate to an empty htt:organizationalEntity in case of an
      * error. If the task name is not present the current task MUST be considered.
+     * In : task name (optional)
+     * Out : potential owners (htt:organizationalEntity)
      */
     public class GetPotentialOwners implements XPathFunction {
+
         public Object evaluate(List args) throws XPathFunctionException {
             GenericHumanRoleDAO potentialOwners = null;
+            TaskDAO taskDAO = evalCtx.getTask();
```

Comment:
Here the evalContext is used only internally. Purpose of the evalContext is holding the task info. We initialise it only in two occasions, after creating a new task or when a request is coming from client api in which the task id is required in the request. So that it cannot be null logically. But it is true theoretically. Should I put a null check here? Thank you for the suggestion.